next/frontend/utils/embeddedFormsHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import { FormDefinition } from 'forms-shared/definitions/formDefinitionTypes'
+import { GetServerSidePropsContext } from 'next/types'
+
+import { environment } from '../../environment'
+
+const map = {
+  olo: environment.embeddedFormsOloOrigins,
+}
+
+export const EMBEDDED_FORM_QUERY_PARAM = 'externa-sluzba'
+export const EMBEDDED_FORM_QUERY_PARAM_TRUE_VALUE = 'true'
+
+const getEmbeddedFormsAllowedOrigins = (formDefinition: FormDefinition) => {
+  if (!formDefinition.embedded) {
+    return null
+  }
+
+  return map[formDefinition.embedded]
+}
+
+export const handleEmbeddedFormRequest = (
+  formDefinition: FormDefinition,
+  context: GetServerSidePropsContext,
+): { success: false; isEmbedded: undefined } | { success: true; isEmbedded: boolean } => {
+  const isEmbeddedQueryParam =
+    context.query[EMBEDDED_FORM_QUERY_PARAM] === EMBEDDED_FORM_QUERY_PARAM_TRUE_VALUE
```

Comment:
the `EMBEDDED_FORM_QUERY_PARAM_TRUE_VALUE` seems a bit cumbersome

any drawbacks to just `EMBEDDED_FORM_QUERY_PARAM in context.query` and

```ts
 const isEmbeddedPostfix = isEmbedded
    ? EMBEDDED_FORM_QUERY_PARAM
    : ''
```

Reply:
I tried this and wanted it to be this way, but Next doesn't work with query params like this unfortunately.